cdap-docs/application-templates/source/etl/templates/sinks/batch/cube.rst

```diff
@@ -42,9 +46,78 @@ cubeFact.timestamp.field is provided).
 **cubeFact.measurements:** Measurements to be extracted from StructuredRecord to be used in CubeFact.
 Provide properties as a JSON Map. For example, to use the 'price' field as a measurement of type gauge,
 and the 'count' field as a measurement of type counter, the property should have the value:
-{"cubeFact.measurement.price":"GAUGE", "cubeFact.measurement.count":"COUNTER"}.
+{"cubeFact.measurement.price":"GAUGE", "cubeFact.measurement.quantity":"COUNTER"}.
 
 .. rubric:: Example
 
-TODO: Fill me out
-
+The following configuration tells to write data into Cube dataset with name "myCube"; provides dataset properties
+(dataset.*) for creating new dataset, if the one with given name doesn't exist; configures measurements to aggregate and
+how to determine timestamp for the facts to be written into Cube::
+
+    {
+        "name": "myCube",
+
+        "dataset.cube.resolutions": "1,60,3600",
+        "dataset.cube.properties": {
+          "dataset.cube.aggregation.byName.dimensions": "name",
+          "dataset.cube.aggregation.byNameByZip.dimensions": "name,zip",
+        },
+
+        "cubeFact.timestamp.field": "ts",
+        "cubeFact.timestamp.format": "MM/dd/yyyy HH:mm:ss",
+        "cubeFact.measurements": {
+          "cubeFact.measurement.price": "GAUGE"
+          "cubeFact.measurement.quantity": "COUNTER"
+        }
+    }
+
+Once data is there, you can now run queries using AbstractCubeHttpHandler (you will need to deploy an application that
```

Comment:
Once the data is there, you could run queries using an AbstractCubeHttpHandler. (You'd need to deploy an application
containing a service using it.) A sample query::